app/dao/notifications_dao.py

```diff
@@ -788,7 +788,7 @@ def letters_missing_from_sending_bucket(seconds_to_subtract):
 
 
 def dao_precompiled_letters_still_pending_virus_check():
-    ninety_minutes_ago = datetime.utcnow() - timedelta(seconds=5400)
+    ninety_minutes_ago = datetime.utcnow() - timedelta(seconds=600)
```

Comment:
variable name will need updating

Reply:
yeah i saw that, will update the content. have we ever had this zendesk ticket get triggered? i cant find any example of it in my inbox

Reply:
It used to get triggered because that's all the task did - open a ticket. But I'm not sure it does now that it only gets opened if the letter can't be automatically fixed

Reply:
@klssmith / @idavidmcdonald i've updated variable names, tests, etc